Typo corrections & printer's error correction

Chairzard opened the pull request:
All changes were taken from the following source: https://archive.org/details/novelsoffyodordo04dost/

- The changes at `@@ -12265,7 +12265,7 @@` (text) and `@@ -14615,7 +14615,7 @@` (html) look like they were a correction to a printer's error made during the transcription (the original source text reads "instance"), but the correction seems to have a typo (insistance should be insistence)?

Please let me know if any changes are needed for this request!

(this is the second/final batch of typos I'll be submitting for this book)

Chairzard commented:
Pulling this out of draft status. Added a final change to the text/html file. Ready for review!

rtonsing reviewed (changes requested):
At line 20133 in the text file, I would like `“villain and tyrant,”` to be joined with the previous paragraph, to match the source.

It would be nice also to center `“began kissing Katia.”`.

Same for the HTML, plus the following paragraph should not be indented.

Otherwise, it all looks good!

Chairzard commented:
> At line 20133 in the text file, I would like `“villain and tyrant,”` to be joined with the previous paragraph, to match the source.
> 
> It would be nice also to center `“began kissing Katia.”`.
> 
> Same for the HTML, plus the following paragraph should not be indented.
> 
> Otherwise, it all looks good!

I made some of these changes. I had a few questions/comments below. Disclaimer:  The world of github/css/html is all pretty new to me so forgive me if any of these questions seem silly!

1. As far as I can tell in the text file, lines 20131 and 20132 are joined; line 20131 is at 72 characters, so I began `“villain` at line 20132. It should be joined in the HTML in lines 23803-23804 as well. Let me know if I'm missing something in my reasoning!
2. Regarding the changes for text file lines 20131-20134, in the source text I linked, there's no comma after `tyrant` so I removed it in my changes. I can re-add the comma after tyrant if needed (I saw you used it with the comma in your reply so I just wanted to double-check).
3. I removed the indent from the paragraph after "Katia" in the HTML file.
4. I am unsure of the best way to center `“began kissing Katia.”` in the HTML. I did see a `poem` class in the css, but I'm not sure if you're looking for that or a new class; let me know (`poem` seems to be used for text that's italicized in the source text, but this line is not italicized).
5. I'm also not sure how to center `“began kissing Katia.”` in the text file. I saw that 2 blank spaces are used when using the poem tag, but see the question about the css class to use above (if using a different class, I'm not sure how many spaces would be needed).

rtonsing commented:
1. OK
2. OK. I might have copied from the wrong place.
3. OK
4. It is common to define in CSS: `.center {text-align: center;}`, then you can use `<p class="center">`.
5. Maybe just have 4 leading spaces, since nothing else is centered in the file.

Chairzard commented:
I've made these changes; please review and let me know if all looks good.
